.travis.yml

```diff
@@ -1,25 +1,48 @@
-dist: trusty
 language: python
-
+install:
+- pip install coverage coveralls
+- python setup.py install
+script:
+- coverage run test.py
+after_success:
+- coveralls
 addons:
   apt:
     packages:
-     - libmecab-dev
-     - mecab
-     - mecab-ipadic-utf8
-     - swig
-
-python:
- - "2.7"
- - "3.5"
- - "3.6"
+    - libmecab-dev
+    - mecab
+    - mecab-ipadic-utf8
+    - swig
 
-install:
- - pip install coverage coveralls
- - python setup.py install
-
-script:
- - coverage run test.py
-
-after_success:
- - coveralls
+matrix:
+  include:
+  - dist: trusty
+    python: '2.7'
+  - dist: trusty
+    python: '3.5'
+  - dist: trusty
+    python: '3.6'
+  - os: osx
+    language: generic
+    env: DEPLOY="true"
+    install: true
+    script: "./ci/build-macos.sh"
+    after_succes: true
+  - service: docker
+    sudo: required
+    env: DEPLOY="true"
+    install: true
+    script: "docker run --rm -v `pwd`:/io quay.io/pypa/manylinux1_x86_64 /io/ci/build-manylinux.sh"
+    after_succes: true
+deploy:
+  provider: releases
+  file_glob: true
+  file: dist/*
+  skip_cleanup: true
+  on:
+    tags: true
+    repo: SamuraiT/mecab-python3
+    condition: "$DEPLOY = true"
+  # Replace appropreate api_key
+  #api_key:
+  #  secure: q11SMpwB2uq7zhH4rreOE/5JO9gfBVBNmOD7uXSTToOAaj5qUSD/TBfI8GYELtalrVGn2O2WCrekpk4WRZ+FdH1jFUmjF/scT3j/3ReFxrgKL4++ll6H2f5pSRTmHIY4aAn4h3ZB0PN/nUqi+qxm6DgC2ptJHhCXs/zb4kFCR3izkuieGBpRndYHlthIv46A86UYw8fHLJPzSYkwYFj75bAims7Y8xTFSLC5rp4EvAQUwu8vr4iI0EgXLsnBTTAUNpoaj/3jCkF08Kp2Ecy9qywJjMtk6VBpAgseN8NDqAQpq9EiW9Gw684yqXaBr9AzIaOtkVuszTrmxjI0gh3K52CosecRgkL66RgUeBpt633ENJiOqsr4JdyWeVEIh3N75212lqHqsK1E8m5jWn0TlwDpgEMGE9GzP/YedIZzjtOVTuK9obLstZhhOAWgPQAiAVyAHe7sF9Tnc5ZR5cIMNFI64hToCQAX0EQ9pPQ1iuKc078JcxlXCJHt41fhkk2zlG48FbXioWbvMsLGV2nsXSwtylu8kw+opH9NVTx1XMxLWkE7QGDeY61NkGHf9ug8mrFQP72Bo64lcUp2uV4rQz5xItXBVhmI0XcJ9KCYYZfgn9is8dondCwNTfqqQvvYTsCZie7dglJjorAxnALZKTMtQwcbYQ5X8Jwut6f+j+I=
```

Comment:
Where did this api key come from? Whose identity is it borrowing?

Reply:
It's mine for test on chezou/mecab-python3. Will replace with `REPLACE-ME`.